debug_scripts/tests/send_validator_logs_test.py

```diff
@@ -0,0 +1,18 @@
+# from typing_extensions import assert_type
+import unittest
+import datetime
+import io
+import sys
+from send_validator_logs import filter_log_file
+
+class test_validator_log_filtering(unittest.TestCase):
+
+    def test_time_filtering(self):
+        start_time = datetime.datetime(2022, 4, 4, 23, 42, 0, 0)
+        end_time = datetime.datetime(2022, 4, 4, 23, 49, 0, 0)
+        output_file_obj = filter_log_file("./tests/data/node0.log", start_time=start_time, end_time=end_time)
```

Comment:
I don't see the file in this PR. did you forget to add it?

Reply:
Yeah, .log was in .gitignore didn't realize. Pushed the file